semver.md

```diff
@@ -102,8 +102,10 @@ intended compatibility requirements as denoted by its associated
 normal version. Examples: 1.0.0-alpha, 1.0.0-alpha.1, 1.0.0-0.3.7,
 1.0.0-x.7.z.92.
 
-1. Build metadata MAY be denoted by appending a plus sign and a series of dot
```

Comment:
So my concern here is that the language used to define build identifiers was pretty much exactly the same as the language used above to describe pre-releases. Why is it more confusing for build identifiers? Should we change the description of pre-release too?